products/gradle.md

```diff
@@ -13,28 +13,74 @@ eolColumn: Bug and Security Fixes
 
 auto:
 -   git: https://github.com/gradle/gradle.git
+    # regex to exclude versions below 3.x for tags with wrong dates see https://github.com/endoflife-date/endoflife.date/pull/3619
+    regex: ^v?(?<major>([3-9]|\d{2,}))\.(?<minor>0|[1-9]\d*)\.?(?<patch>0|[1-9]\d*)?$
 
+# support(x) = releaseDate(x+1)
+# eol(x) = releaseDate(x+2)
```

Comment:
Not strictly related to this PR, but looking at https://docs.gradle.org/current/userguide/feature_lifecycle.html#eol_support the support and EOL looks clearly defined now. The description states that _The support and EOL policy is not clearly defined_, so the description must be updated.

@BiNZGi, If you want to take care of it that would be great. Else let's open an issue to not forget.